taskutil.h

```diff
@@ -0,0 +1,5 @@
+#ifndef __TASKUTIL_H__
+#define __TASKUTIL_H__
+#include <string>
```

Comment:
Please keep code formatting consistent (usage of vertical whitespace should be similar across files).